be/src/exec/spill/mem_table.cpp

```diff
@@ -14,19 +14,74 @@
 
 #include "exec/spill/mem_table.h"
 
+#include <glog/logging.h>
+
+#include <memory>
+
 #include "column/chunk.h"
 #include "column/vectorized_fwd.h"
 #include "exec/chunks_sorter.h"
+#include "exec/spill/input_stream.h"
 #include "runtime/current_thread.h"
 
-namespace starrocks {
-namespace spill {
+namespace starrocks::spill {
+class RawChunkInputStream final : public SpillInputStream {
+public:
+    RawChunkInputStream(std::vector<ChunkPtr> chunks) : _chunks(std::move(chunks)) {}
+    StatusOr<ChunkUniquePtr> get_next(SerdeContext& ctx) override;
+
+    bool is_ready() override { return true; };
+    void close() override{};
+
+    bool enable_prefetch() const override { return true; }
+
+    Status prefetch(SerdeContext& ctx) override {
```

Comment:
what does the prefetch mean? Early stop?

Reply:
fetch a chunk in IO Thread. see detail in SpilledInputStream